workflow/scripts/run_readii.py

```
@@ -17,28 +17,67 @@
+    setVerbosity(50)
+
     # check if file already exists
     if region and transform:
         sample_result_file_name = f"{region}_{transform}_features.csv"
     else:
         sample_result_file_name = f"full_original_features.csv"
     
+    # Set up sample directory to save output files in
     complete_out_path = sample_dir_path / sample_result_file_name 
     if complete_out_path.exists() and (not overwrite):
         print(f"Features already extracted for: {complete_out_path.stem}")
```

Ruff (0.8.2)

`print` found

Remove `print`

(T201)